packages/core/lib/messages/baseMessageSchemas.ts

```diff
@@ -0,0 +1,25 @@
+import z from 'zod'
+
+export const BASE_MESSAGE_SCHEMA = z.object({
```

Comment:
do we have plans to use `BASE_MESSAGE_SCHEMA` without metadata? If no, I would make it private

Reply:
This is an opensource lib, so it would be great to leave space for people to use it with whatever data structure that they have or want to have. BASE_MESSAGE_SCHEMA offers the minimum subset that message-queue-toolkit itself relies on.